server/routes/alumni.js

```diff
@@ -14,16 +19,74 @@ router.post('/', async (req, res, next) => {
         const email = req.body.email;
         const name = req.body.name;
         const gradYear = parseInt(req.body.graduationYear);
-        const location = req.body.location;
-        const profession = req.body.jobTitle;
-        const company = req.body.company;
-        const college = req.body.college;
+        const country = req.body.country;
+        const city = req.body.city;
         const password = req.body.password;
         // TODO: need to add timeZone in frontend request
         const availabilities = [];
         const timeZone = req.body.timeZone;
         const zoomLink = req.body.zoomLink;
+        const schoolId = req.body.schoolId;
 
+        // find or create College
+        const newCollege = req.body.newCollege
```

Comment:
For pooled indexable fields, both new* and existing*Id are sent as part of response, and a `findOrCreate` model is used to add new entries to collection